src/gam/gapi/directory/cros.py

```diff
@@ -15,11 +16,20 @@
 from gam import utils
 
 
+def _display_cros_command_result(cd, device_id, command_id, times_to_check_status):
+    for _ in range(0, times_to_check_status):
+        time.sleep(2)
+        result = gapi.call(cd.customer().devices().chromeos().commands(), 'get',
+            customerId=GC_Values[GC_CUSTOMER_ID], deviceId=device_id,
+            commandId=command_id)
+        display.print_json(result)
+        if result.get('state') in {'EXPIRED', 'CANCELLED', 'EXECUTED_BY_CLIENT'}:
```

Comment:
```
final_states = ['EXPIRED', 'CANCELLED', 'EXECUTED_BY_CLIENT']
if result.get('state') in final_states:
....
```
it's easier to understand what the code is doing here if we declare what we believe are final states ahead of the if statement.